src/python/ensembl/io/genomio/assembly/status.py

```diff
@@ -0,0 +1,469 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Obtain and record the assembly status for a set of INSDC accession(s) using NCBI's datasets CLI tool."""
+
+__all__ = [
+    "check_parameterization",
+    "datasets_asm_reports",
+    "extract_assembly_metadata",
+    "fetch_accessions_from_cores",
+    "generate_report_tsv",
+    "resolve_query_type",
+    "singularity_image_setter",
+]
+
+import csv
+import json
+import logging
+import os
+from os import PathLike
+from pathlib import Path
+import re
+import sys
+from typing import Dict, Tuple, Union, List
```

Comment:
```suggestion
from typing import Dict, List, Tuple, Union
```
Two minor tweaks to make myself and `pylint` happy.